connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/PostgresIntegrationSuite.scala

```diff
@@ -252,6 +253,17 @@ class PostgresIntegrationSuite extends DockerJDBCIntegrationV2Suite with V2JDBCT
     }
   }
 
+  test("SPARK-49695: Postgres fix xor push-down") {
+    val df = spark.sql(s"select dept, name from $catalogName.employee where dept ^ 6 = 0")
+    val rows = df.collect()
+    assert(df.queryExecution.sparkPlan.collectFirst {
+      case f: FilterExec => f
+    }.isEmpty)
+    assert(rows.length == 1)
+    assert(rows(0).getInt(0) === 6)
+    assert(rows(0).getString(1) === "jen")
```

Comment:
Could you use `checkAnswer`, please.

Reply:
This file has mainly these asserts, wanted to make it in line with other tests